### PR TITLE
docs: Add audit logs docs

### LINKED ISSUE #3418: Add documentation for Audit page
As a user viewing the audit page, I would like to be able to link to documentation that tells me what sort of events are tracked. 

[Audit RFC](https://www.notion.so/coderhq/v2-Enterprise-Audit-Logging-a6358d50a453413eae6441e689f023ac)

[Audit Mocks](https://www.figma.com/file/oy7echtkBVRuOCMBVSrr4T/Auditing?node-id=0%3A1)

## AC

- [ ] Create a documentation page for the Audit Log. This page should include what sort of events we track.
- [ ] Amend the Audit header tooltip such that it matches the mocks and provides a link to the new documentation page ("Events we track" in the below screenshot): 
Screen Shot 2022-08-08 at 4 48 02 PM

- [ ] Add a test for the above work.

## Definition of Done
Above AC is completed.

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
Closes #3418 

@bpmct I would appreciate your help on this

### CODE REVIEW

File: docs/admin/audit-logs.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Audit Logs
+
+Audit logs is an enterprise feature that allows **Admins** and **Auditors** to monitoring what is happening on their deployment.
```
Comment (jsjoeio) [inline docs/admin/audit-logs.md:3]:
```suggestion
This is an enterprise feature that allows **Admins** and **Auditors** to monitor what is happening in their deployment.
```

File: docs/admin/audit-logs.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Tracked Events
+
+The following events are tracked by this feature:
```
Comment (jsjoeio) [inline docs/admin/audit-logs.md:7]:
```suggestion
The following events are tracked:
```

File: docs/admin/audit-logs.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Create
+- Update
+- Delete
```
Comment (jsjoeio) [inline docs/admin/audit-logs.md:11]: Can we elaborate? Is it any create event? (user, workspace template)

File: docs/admin/audit-logs.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+For the following resources:
+
+- GitSSHKey
+- Template
+- TemplateVersion
+- Workspace
+- APIKey
+- User
```
Comment (jsjoeio) [inline docs/admin/audit-logs.md:20]: Ooooo I see it now... Hmm, I wonder if we can structure this better. Maybe something like:

This feature tracks create, update and delete events for the following resources:
- list

File: docs/admin/audit-logs.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+## Enabling this feature
+
+To enable this feature an Admin needs to purchase an enterprise license.
```
Comment (jsjoeio) [inline docs/admin/audit-logs.md:24]: And then what? It's automatically enabled?

Maybe like 
```suggestion
This feature is autoenabled for all enterprise deployments. An Admin can purchase a license [here](link)
```

**Review by jsjoeio — CHANGES_REQUESTED** [review]
Nice work Bruno! I added some suggestions. Let me know if you want more help!

File: docs/admin/audit-logs.md (hunk 5 above)
Comment (BrunoQuaresma) [reply docs/admin/audit-logs.md:24]: Yeah, we don't have a link yet :(

Comment (jsjoeio) [reply docs/admin/audit-logs.md:24]: Ah gotcha. Yeah whatever you think!

File: docs/admin/audit-logs.md (hunk 4 above)
Comment (BrunoQuaresma) [reply docs/admin/audit-logs.md:20]: Makes sense.

**Review by jsjoeio — APPROVED** [review]
Great idea adding the contact link with the note 🔥

**Pull request merged**